.github/workflows/update_sdk_methods.py

```diff
@@ -19,8 +19,8 @@
 ##   type = ["array", "of", "resources"]
 ## You can use the target_resources positional parameter to refine this
 ## at runtime if desired:
-components = ["arm", "base", "board", "camera", "encoder", "gantry", "generic_component", "gripper",
-              "input_controller", "motor", "movement_sensor", "power_sensor", "sensor", "servo"]
+components = ["arm", "base", "board", "button", "camera", "encoder", "gantry", "generic_component", "gripper",
+              "input_controller", "motor", "movement_sensor", "power_sensor", "sensor", "servo", "switch"]
```

Comment:
Button and Switch aren't implemented in python yet.

Reply:
True. I'm not adding any generated Python examples/descriptions that don't exist yet, just adding it to this list so the automation knows to scrape for these resources for all SDKs.

Reply:
(I haven't added brand new APIs much using this script before, but it seems to have had that effect! :D )